dataset.py

```diff
@@ -18,24 +18,37 @@
 class Dataset():
     """The Google Analytics dataset."""
 
-    def __init__(self, debug=False):
+    def __init__(self, debug=False, skip_rows=False):
         """Load the data from disk.
 
         Args:
             debug (bool): An option to choose whether to load all
-              data.  If 'debug' is true, program will only read 100 rows
+              data.  If 'debug' is true, program will only read 1000 rows
               data from the csv file.
               However, one thing to pay attention is that if you load
               less data, the shape of DF is wrong, because some
               columns daon't have any data until you read many many
               rows.
+            skip_rows (bool): An option to load an evenly distributed
+              sample of the dataset. If 'debug' is true, _approximately_
+              1000 rows will be read from the csv file, but taken every
+              _NUM_SKIP_ROWS_TRAIN and _NUM_SKIP_ROWS_TEST rows instead
+              of just the first 1000 rows.
+              NOTE: Automatically False if debug = False
```

Comment:
Instead of leaving a warning, could you raise a `ValueError` if `debug` is false and `skip_rows` is true?